src/iuse_actor.cpp

```diff
@@ -112,9 +112,7 @@ long iuse_transform::use(player *p, item *it, bool t, const tripoint &pos ) cons
         // "created" right now and give the content the current time as birthday
         it->convert( container_id );
         it->unset_curammo();
-        it->charges = -1;
-        it->contents.push_back(item(target_id, calendar::turn));
-        target = &it->contents.back();
+        target = &it->emplace_in( target_id, calendar::turn, -1 );
```

Comment:
`charges` is set to the magic value of `-1` to avoid any possible behavioral changes. Such uses will need further identifying as part of a separate PR